third_party/llm_engine/llm_engine.bzl

```diff
@@ -128,8 +128,6 @@ cmake(
         }}),
     env = {{
         "OpenVINO_DIR": "{OpenVINO_DIR}",
-        "HTTP_PROXY": "{http_proxy}",
```

Comment:
Do not remove this.

Reply:
libicu does not build with multiple envs